go.mod

```diff
@@ -121,4 +121,4 @@ require (
 )
 
 // add status code to HTTP request metrics, see https://github.com/open-telemetry/opentelemetry-go-contrib/pull/771
-replace go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => github.com/textileio/opentelemetry-go-contrib/instrumentation/net/http/otelhttp v0.26.1-0.20220407173812-deccbe316725
+replace go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => github.com/textileio/opentelemetry-go-contrib/instrumentation/net/http/otelhttp v0.26.1-0.20230224190607-b4c579e82b23
```

Comment:
I had to update this library that we maintain as well. Here's the [patch](https://github.com/textileio/opentelemetry-go-contrib/commits/v1.14.0-patched).